docs/rfc/rfc-1.adoc

```diff
@@ -225,9 +234,17 @@ conditions for the btc fee and then off-chain see which deposits *would be*
 swept if we *were* to sweep. If the sum of those deposits exceeds the
 `sweep_max_btc` parameter, then we should initiate a sweep.
 
-After submitting the transaction, it will cost some amount of bitcoin based on
-what miners are charging for the bitcoin fee. This fee comes out of the unswept
-deposits. When the transaction clears, and the information has made its way
+The sweeping transaction will cost some amount of bitcoin based on what miners
+are charging for the bitcoin fee in the current market conditions. The fee is
+paid proportionally from the unswept deposits. Once the transaction is
```

Comment:
Proportionally to the deposit amount? Meaning the bigger deposit, the bigger fee? Do we want that?
My understanding is that the fee depends on number of UTXOs that are being swept (and also market conditions), so it seems fair to me to split the fee equally between all UTXOs.

Reply:
extremely good catch